recipes/forest_lite/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "forest_lite" %}
+{% set version = "0.16.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/MetOffice/forest-lite/archive/v{{ version }}.tar.gz
+  sha256: 24103013a84aad2303986186ce6d328dcabf3e659a81ed88919de9328f12ecfe
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.7
```

Comment:
Upstream says >=3.6, not that is matters much here but if you want to reach 3.6 users you may consider changing that. See https://github.com/MetOffice/forest-lite/blob/master/setup.py#L69

Reply:
@ocefpaf well spotted, I'll change it to `3.6`